packages/core/src/ceramic.ts

```diff
@@ -655,6 +663,45 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
     )
   }
 
+  /**
+   * Waits for Model used to publish NodeMetrics to be available locally.
+   * Since we subscribe to the metamodel at startup, so long as some connected node on the network
+   * has the model, it should eventually be available locally.
+   * @param model
+   * @returns whether or not the model was loaded successfully
+   */
+  async _waitForMetricsModel(model: StreamID): Promise<boolean> {
+    const maxWaitDuration = 1000 * 15 // 15 seconds
+    const retryInterval = 100
```

Comment:
i think its going to take more than .1 sec?  a cascading backoff might be nice.  but this is fine for now.